packages/toolkit/src/query/tests/buildHooks.test.tsx

```diff
@@ -46,7 +47,7 @@ interface Item {
 
 const api = createApi({
   baseQuery: async (arg: any) => {
-    await waitMs(150)
+    await waitForFakeTimer(20)
```

Comment:
Ok, the timeout was 150ms, and I reduced it to 10 to make tests faster, I experimentally chose this value running tests on my laptop. In the latest push I enlarged it to 20, let’s see if it helped with tests in CI

Reply:
@markerikson could you start the CI again please? If it fails, I’ll put the 150ms back

Reply:
Okay I see. Pushed 150ms back 🤞

Reply:
Thanks, looks like the check is green now.